### DISCUSSION TITLE
Collection not downloading

### DISCUSSION THREAD

**Discussion by jchow234** [Avnsx/fansly#141] [Q&A]
Hey, I'm trying to download some collections I already unlocked for a creator I previously subscribed to (currently not subscribed). I have updated the config auth and user agent, and mode to Collections. The exe window just suddenly closes on its own but it doesn't seem to be a config issue on my part. Is this issue because I am no longer subscribed? 

![Screenshot 2023-05-31 012054](https://github.com/Avnsx/fansly-downloader/assets/51731014/df00eb99-2e52-4927-a525-fd642f7b48bf)

**Comment by Avnsx** [comment]
If you see stuff within https://fansly.com/collection it should be able to download that.
Your output looks like it can't find anything within the collection linked to your account, fansly just sends a empty json response.
Are you sure fansly-downloader is connected to the right account? Double check the auth token.

**Reply by jchow234** [reply]
Yeah I've tried it again, and I used both chrome and firefox with no success. Is it important to be subscribed? I can clearly see the downloads I want with the link to collections that you posted

**Reply by Avnsx** [reply]
I think I may know why you're having this issue.

If you have knowledege of python (or just read the [set-up tutorial for the python version](https://github.com/Avnsx/fansly-downloader#%EF%B8%8F-set-up)), would you please install the python "Source Code" from previous [version 0.4](https://github.com/Avnsx/fansly-downloader/releases/tag/v0.4), then try to use version 0.4 to download any content from Messages?

If that previous version 0.4 makes it impossible for you to download content from Messages, then it's also the same reason you're not able to download collections in version 0.4.1.

If that case above is true, it's most likely because of the issue with ``limit``, I had previously described [here](https://github.com/Avnsx/fansly-downloader/issues/104#issuecomment-1595178750) because I applied the ``limit`` variable change to messages and collections, but then only reverted it for messages, but kept it for collections.

**Reply by jchow234** [reply]
![image](https://github.com/Avnsx/fansly-downloader/assets/51731014/95cb089f-96ee-4b44-8112-df108ca4ed66)
This is the result I get when I run it via terminal python with version 0.4, I get a same result when I alter the 'limit' variable. 

I'm not super good with terminal/powershell so it's possible I fcked it up. I did try downloading from the same creator on normal mode via the .exe file again, it technically ran but it did not download anything (even after I removed all the previously downloaded content). I am kind of stumped lol

**Reply by Avnsx** [reply]
It was already obvious to me that you would not be able to download collections in version 0.4 either, becasue it's the same code that is handling collections in both versions.

Which is the reason why I was specifically asking if you could download content from **Messages** using version 0.4.
Set "download_mode" to "Messages" and try if you can download anything, from anyone.

**Reply by jchow234** [reply]
Hey sorry, had a super busy week, but I just tried via the python version and yes, I can download messages from the creator i am currently trying on for Collections - does this mean you have an idea on how to solve my issue?
